### DISCUSSION TITLE
How can I have a real-time forked mainnet?

### DISCUSSION THREAD

**Discussion by 0xAWM** [ConsenSys-archive/truffle#5356] [General Q&A]
In [Simulate Live Networks with Forked Sandboxes](https://trufflesuite.com/blog/sandbox-forking-with-truffle-teams/) we know that that's easy to fork a mainnet from a given block number. 
I want to have a real-time forked mainnet. I mean that when I send a transaction to the forked network, it's always communicate with the rpc in the latest block rather that the state when I run the fork command.

**Comment by leeftk** [comment]
Hey @mathgo thanks for bringing this up. It's actually a very common request, but from a technical perspective it's highly improbable to implement this. We are looking at ways that we could do something similar to this in the future but at the moment Ganache doesn't support this feature. Hope this helps answer your question!
